data/wanderers start.txt

```diff
@@ -2443,8 +2443,23 @@ mission "Wanderers Ap'arak 3: Pug"
 		ship "Pug Arfecta" "Hork Ekekek Ptui"
 		ship "Pug Arfecta" "Graak Sput Kchoo"
 	on complete
+		dialog Shortly after destroying the Pug ships, you received a message from Sobari Tele'ek: "We have received [news, reports] that you are destroying the ships of our Keeper [friends, allies] in the Ap'arak system. This is [unacceptable, intolerable]. From now on, you are no longer [welcome, allowed] in our [territory, space]."
 		"reputation: Wanderer" <?= -1
 
+mission "Wanderers Ap'arak Pug Patrol Increased"
```

Comment:
I'm not merging this mission yet, so please remove it

Reply:
Removed.